rpc/tests/share.rs

```diff
@@ -63,10 +63,20 @@ async fn test_get_shares_by_namespace_wrong_ns(client: &HttpClient) {
         .unwrap()
         .dah;
 
-    client
-        .share_get_shares_by_namespace(&dah, random_ns())
+    let random_ns = random_ns();
+    let ns_shares = client
+        .share_get_shares_by_namespace(&dah, random_ns)
         .await
-        .unwrap_err();
+        .unwrap();
+
+    if !ns_shares.rows.is_empty() {
```

Comment:
is this valid? if yes add a comment with brief explanation.

Reply:
it is, explained